index.html

```diff
@@ -54,8 +57,189 @@ <h3>I'm a Software Developer</h3>
+      </div>
+      <div class="links">
+        <ul>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <div class="buttons">
+        <a href="#">See project</a>
+      </div>
+    </section>
+    <section class="img_p_holder">
+      <div class="mp_stories">
+        <h2>Professional Art<br> Printing Data</h2>
+        <p>A daily selection of selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard.</p>
+      </div>
+      <div class="links">
+        <ul>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <div class="buttons">
+        <a href="#">See project</a>
+      </div>
+    </section>
+    <section class="img_p_holder">
+      <div class="mp_stories">
+        <h2>Professional Art<br> Printing Data</h2>
+        <p>A daily selection of selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard.</p>
+      </div>
+      <div class="links">
+        <ul>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <div class="buttons">
+        <a href="#">See project</a>
+      </div>
+    </section>
+    <section class="img_p_holder">
+      <div class="mp_stories">
+        <h2>Professional Art<br> Printing Data</h2>
+        <p>A daily selection of selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard.</p>
+      </div>
+      <div class="links">
+        <ul>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <div class="buttons">
+        <a href="#">See project</a>
+      </div>
+    </section>
+    <section class="img_p_holder">
+      <div class="mp_stories">
+        <h2>Professional Art<br> Printing Data</h2>
+        <p>A daily selection of selection of privately personalized reads; no accounts or sign-ups required. has been
+          the industry's standard.</p>
+      </div>
+      <div class="links">
+        <ul>
+          <li>HTML</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <div class="buttons">
+        <a href="#">See project</a>
+      </div>
+    </section>
+    <section class="about">
+      <div class="bottom_line">
+        <div class="about_summary padding_lr">
+          <h1>About me</h1>
+          <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact
+            me.
+            Get my resume
+          </p>
+        </div>
+        <div class="buttons">
+          <a href="#">Get my resume</a>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] There are two horizontal lines after the design for the About section. Kindly remove the bottom horizontal line so that your design is identical to that of the Figma design. I highlighted the line to remove with orange ink in the screenshot below👍: 

| Your Design  | Figma Design |
| ------------- | ------------- |
| ![Screenshot_2](https://github.com/wacoo/Portfolio-Project/assets/77106962/f356d650-fba8-48b7-a89e-16395506efe0) | ![image](https://github.com/wacoo/Portfolio-Project/assets/77106962/dd239b95-2719-4dfd-8c1e-8d282617f64b)
 |